.gitlab-ci.yml

```diff
@@ -68,6 +68,7 @@ stages:
         -e KFP_RUN_URL_PREFIX=$KFP_RUN_URL_PREFIX 
         -e KFP_SDK_NAMESPACE=$KFP_SDK_NAMESPACE 
         -e USER=$GITLAB_USER_EMAIL
+        -e ZODIAC_SOURCE=CICD
```

Comment:
`CICD` looks fine to me, but how would you feel about `integration_test`?

Reply:
Just had a discussion with @alexlatchford , and we agreed that `source` is more a customer usage scenario, and would need more thought behind it (ex: source might end up being a Flow trigger, or maybe it is "trigger" instead of "source"? etc)

Reply:
So I reverted and removed `source` and kept this PR to zodiac `owner` only.